libs/blocks/media/media.css

```diff
@@ -1,4 +1,6 @@
 /* media - consonant v5.1 */
+@import '../../styles/rounded-corner.css';
```

Comment:
Instead of importing for all media blocks, what about checking for the existance of a `rounded-corners` class and only import rounded-corner.css (via loadStyle) when needed.

Reply:
Would it be fine to dynamically loadStyle in every js file that needs the styling? Right now is needed only for media and Brick block, but it might be extended to more blocks .The css file is not big, thus directly imported it on block's css file.
@chrischrischris @rgclayton @salonijain3